src/Voron/Impl/LowLevelTransaction.cs

```diff
@@ -1112,6 +1112,21 @@ public void RetrieveCommitStats(out CommitStats stats)
             _requestedCommitStats = stats = new CommitStats();
         }
 
+        public string GetTxState()
+        {
+            switch (_disposed)
```

Comment:
Just return the enum value and let the `.ToString()` handle this

Reply:
done